terra_notebook_utils/drs.py

```diff
@@ -230,6 +230,10 @@ def _resolve_bucket_target(url: str, info: DRSInfo) -> Tuple[str, str]:
     return bucket_name, key
 
 def _resolve_local_target(filepath: str, info: DRSInfo) -> str:
+    # Checking for a file also prevents a batch_copy from overwriting
+    # the same file over and over again.
+    if os.path.isfile(filepath):
+        raise FileExistsError(filepath)
```

Comment:
Could this error message say something like `Attempting to batch copy {info.name} to {filepath}, but this path would be overwritten, aborting.`

Reply:
I reworded and removed `batch` from the message since this applies to all copies, but otherwise :+1: